Formula/lotus-miner.rb

```diff
@@ -20,8 +20,8 @@ def install
   on_linux do
     on_intel do
       if Hardware::CPU.is_64_bit?
-        url "https://github.com/filecoin-project/lotus/releases/download/v1.28.3/lotus-miner_v1.28.3_linux_amd64.tar.gz"
-        sha256 "some"
+        url "https://github.com/filecoin-project/lotus/releases/download/miner/v1.32.1/lotus-miner_v1.32.1_linux_amd64_v1.tar.gz"
```

Comment:
I don't know about Homebrew forumals, but wanted to dobule check that `on_intel` we should should provide an `amd64` link.